### DISCUSSION TITLE
osulazer cant opened

### DISCUSSION THREAD

**Discussion by yukari2** [ppy/osu#19414] [Q&A]
i cant find the osu. no screen and no sound. but the background have osulazer.exe
https://s1.328888.xyz/2022/07/27/DJElh.png
https://s1.328888.xyz/2022/07/27/DJLan.png

sry my poor English :-(

**Comment by frenzibyte** [comment]
https://github.com/ppy/osu/discussions/15188, https://github.com/ppy/osu/discussions/16869, and please state your operating system.

**Reply by yukari2** [reply]
[logs.zip](https://github.com/ppy/osu/files/9197430/logs.zip)
![VZY09FHPXB7V2@N}P}@K(4E](https://user-images.githubusercontent.com/69705938/181224147-146deb7c-5df9-4bb5-90a1-d74fdf30478b.png)
